### DISCUSSION TITLE
Feature discussion "select time range around this line"

### DISCUSSION THREAD

**Discussion by michaelkebe** [grafana/grafana#55211] [Observability]
I am using the grafana explore view to navigate through the logs of loki.

I start with a query like `{host="some.host.name",unit="some.service"} |= "ALARM"`.

If there are a resulting lines, I usually want to see the context around that line, but without the `|= "ALARM"` filter.

Is there any way to speed this up?

Entering the time range is tedious.
It would be great to have a "select time range around this line" maybe with a selectable duration (+/- 1,5,10,60 minutes).

What do you think? Any other suggestions?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
